server/pyproject.toml

```diff
@@ -28,9 +28,6 @@ pyyaml = "^6.0.1"
 passlib = "^1.7.4"
 bcrypt = "3.1.7"
 python-dotenv = "^1.0.1"
-psycopg2 = "^2.9.9"
```

Comment:
Why did you removed them from here?

Reply:
Bymistake it would have been removed, added back